discover/index.html

```diff
@@ -27,72 +27,72 @@ <h2 class="text-left"><i class="fas fa-fw fa-plane"></i> Getting Started</h2>
 </p>
 <h3 class="text-left"><i class="far fa-fw fa-keyboard"></i> Sign Up</h3>
 <h5>Choose the Right Form</h5>
-<p>To obtain access to Discover you will need to fill out one of the following forms <strong>(A or B)</strong></p>
+<p>To obtain access to Discover, you will need to fill out one of the following forms <strong>(A or B)</strong>:</p>
 <ol type="A">
-  <li>If your organization is one of <strong>Utah’s cities, counties, special districts, state agencies, K-12/Higher education, tribes</strong> or are a <strong>contractor and formal partner</strong> of one of these organizations you are covered by our license agreement to access imagery provided by Google on Discover. Please fill out and understand the <a href="https://docs.google.com/a/utah.gov/forms/d/e/1FAIpQLScL5uUQIvw7op_ZcF4bijxcoOMGhNF0MXwJNGqSXS6IbjbKhA/viewform">Organizational Usage Agreement Form</a> to obtain access. To read more about the license visit the <a href="{{ "/discover/license/" | prepend: site.baseurl }}">Google Imagery License</a> page.
-  <li>If you are <strong>not</strong> part of one of the organizations listed above you should fill out the <a href="https://docs.google.com/a/utah.gov/forms/d/e/1FAIpQLScvASb37-R9WeFHNUsbIYEcVzQ_ceT__G4PZUaCx_xZxTuEpA/viewform">Discover Server Access</a> form to get access to all of the non-licensed base maps and imagery services.
+  <li>If you're requesting access on behalf of one of Utah’s: <strong>cities, counties, special districts, state agencies, K-12 or higher education institutions, or tribal entities</strong>, or if you are a <strong>contractor and formal partner</strong> of one of these organizations, you are covered by our license agreement to access imagery provided by Google on Discover. In this case, please read and complete the <a href="https://docs.google.com/a/utah.gov/forms/d/e/1FAIpQLScL5uUQIvw7op_ZcF4bijxcoOMGhNF0MXwJNGqSXS6IbjbKhA/viewform">Organizational Usage Agreement Form</a> to obtain access. You can read more about our Google Imagery license <a href="{{ "/discover/license/" | prepend: site.baseurl }}">here</a>.
```

Comment:
should we be consistent with `you're` and `you are` or is it good to mix it up?

Reply:
In general, I think it just depends on the context--some of it is just readability and what sounds best. But that can definitely be subjective too! "You're" can be a bit more conversational. So it just depends on what we're going for. You can make a call if you want :) Whatever sounds best to you!

Reply:
Interesting! And not super important. I would choose expanded over contracted.